src/screens/reader/components/WebViewReader.tsx

```diff
@@ -124,9 +127,39 @@ const WebViewReader: FC<WebViewReaderProps> = props => {
       subscription.remove();
       mmkvListener.remove();
     };
-  });
+  }, []);
+  const debugging = `
+     // Debug
+     console = new Object();
+     console.log = function(log) {
+      reader.post({"type": "console", "msg": log});
+     };
+     console.debug = console.log;
+     console.info = console.log;
+     console.warn = console.log;
+     console.error = console.log;
+     `;
+  //@ts-ignore
+  const onMessage = payload => {
+    let dataPayload;
+    try {
+      dataPayload = JSON.parse(payload.nativeEvent.data);
+    } catch (e) {}
+
+    if (dataPayload) {
+      if (dataPayload.type === 'console') {
+        // eslint-disable-next-line no-console
+        console.info(`[Console] ${JSON.stringify(dataPayload.msg, null, 2)}`);
+      } else if (false) {
+        // eslint-disable-next-line no-console
+        if (dataPayload.type !== 'save') console.log(dataPayload);
+      }
```

Comment:
tbh logging via React Native for Webview is not an effecient way. Class objects wouldnt be logged.
If someone want to work in Reader, they can create a html with fake data and all reader js/css then open it in browser, which is eaiser and faster way.

Reply:
Of course this isn't a good way. I just implemented this because of things that behave a bit different in webview, than in the browser. E. g. On opening a new chapter, the chapter width isn't always the same at first. Probably just because of slower calculations but nevertheless a bug I haven't encountered on my browser.